src/pages/en/reference/cli-reference.md

```diff
@@ -167,6 +168,16 @@ Specifies the path to the config file relative to the project root. Defaults to
 astro --config config/astro.config.mjs dev
 ```
 
+### `--site`
```

Comment:
Do these need to indicate the arguments they get passed like we do for `--root` and `--config`?

```suggestion
### `--site <url>`
```